package.json

```diff
@@ -216,7 +216,7 @@
     "test": "tap",
     "test-all": "npm run test --if-present --workspaces --include-workspace-root",
     "snap": "tap",
-    "postsnap": "make -s docs",
+    "docs": "npm run build -w docs",
```

Comment:
Another spot where we don't have to run the docs command since they are all ignored and generated.